TOCSharp/TOCClientSettings.cs

```diff
@@ -6,5 +6,6 @@ public class TOCClientSettings
         public ushort Port { get; set; } = FLAPConnection.DEFAULT_PORT;
         public string ClientName { get; set; } = "TOCSharp";
         public bool DebugMode { get; set; } = false;
+        public uint KeepAliveInterval { get; set; } = 120000;
```

Comment:
Use TimeSpan for the interval, makes it easier to pass on Task.Delay

Reply:
Addressed in latest commit.